compiler/common-artifacts/exclude.lst

```diff
@@ -31,8 +31,6 @@ tcgenerate(BatchMatMulV2_000)
 tcgenerate(BatchMatMulV2_001)
 tcgenerate(BatchToSpaceND_000)
 tcgenerate(BroadcastTo_000) # luci-interpreter doesn't support custom operator
-tcgenerate(BroadcastTo_001)
-tcgenerate(BroadcastTo_002)
```

Comment:
At first, I put two of them in the `exclude.lst`, `BroadcastTo_001` and `BroadcastTo_002`, with a plan to add them to the recipe. 
but, it was discussed [here - PR](https://github.com/Samsung/ONE/pull/12012) that only the `BroadcastTo_001` recipe file was needed. 
So `BroadcastTo_001` is erased to **enable** BroadcastTo, and `BroadcastTo_00`2 is erased because there is no recipe file.